nilearn/__init__.py

```diff
@@ -9,66 +9,18 @@
 See http://nilearn.github.com for complete documentation.
 """
 
-from version import __version__
```

Comment:
Please don't remove __version__. It is very handy to figure out which version of the package you are running for example, via:

```python
import nilearn
print nilearn.__version__
```

to figure out the version of your package.